Help page updates for v4.1 (#1433)
broadinstitute/gnomad-browser: browser/help/topics/dataset-selection.md

Note that gnomAD v4.1 has a single subset (non-UK Biobank), but frequencies for this subset are not displayed on the browser and are only available in our downloadable files.

The following subsets from previous versions are currently available, but access to these subsets will change in 2024. Those subsets that will be continued to be supported in the browser and available on our downloads page are indicated in bold, those that will be moved to download only in early 2024 are in italics.

### Short variants

#### gnomAD v4.1

- **gnomAD v4.1**

> - gnomAD v4.1 (non-UK Biobank): Only samples that are not present in the UKBB 455,000 exome release. The allele counts in this subset can thus be added to those of UKBB to enable federated use of both datasets. ([Download only](/downloads#v4))

Author: ```suggestion
- gnomAD v4.1 (non-UK Biobank): Only samples that are not present in the UKBB 455,000 exome release. The allele counts in this subset can thus be added to those of UKBB to enable federated use of both datasets.
```

> 

Author: ```suggestion

#### gnomAD v4.1

**gnomAD v4.0**
- gnomAD v4.0 (non-UK Biobank): Only samples that are not present in the UKBB 455,000 exome release. The allele counts in this subset can thus be added to those of UKBB to enable federated use of both datasets. ([Download only](/downloads#v4))
```

> Note that gnomAD v4.1 has a single subset (non-UK Biobank), but frequencies for this subset are not displayed on the browser and are only available in our downloadable files.

Reviewer: How about here if they will be available in Browser? @ch-kr

Author: we could add a note, but we don't have any notes for other displayed subsets, e.g.
> gnomAD v3.1.2 (non-cancer): Only samples from individuals who were not ascertained for having cancer in a cancer study.

though I'll will add a note now to make this clearer this only impacts the exomes

Author: oh I can't read, will update this also

> Note that gnomAD v4.1 has a single subset (non-UK Biobank), but frequencies for this subset are not displayed on the browser and are only available in our downloadable files.

Author: ```suggestion
Note that gnomAD v4.1 has a single subset (non-UK Biobank). This subset only impacts the gnomAD exomes (does not impact genome frequencies).
```

Reviewer: Could you make it genomes?

Author: done

The following subsets from previous versions are currently available, but access to these subsets will change in 2024. Those subsets that will be continued to be supported in the browser and available on our downloads page are indicated in bold, those that will be moved to download only in early 2024 are in italics.

### Short variants

#### gnomAD v4.1

- **gnomAD v4.1**

> - gnomAD v4.1 (non-UK Biobank): Only samples that are not present in the UKBB 455,000 exome release. The allele counts in this subset can thus be added to those of UKBB to enable federated use of both datasets.

Author: ```suggestion
- gnomAD v4.1 (non-UK Biobank exomes): Only samples that are not present in the UKBB 455,000 exome release. The allele counts in this subset can thus be added to those of UKBB to enable federated use of both datasets.
```
